source/fundamentals/crud/write-operations/embedded-arrays.txt

```diff
@@ -173,92 +189,125 @@ that matches your query, use the filtered positional operator
 The filtered positional operator ``$[<identifier>]`` specifies the
 matching array elements in the update document. To identify which array
 elements to match, pair this operator with ``<identifier>`` in an
-``options.arrayFilters`` object.
-
-The ``<identifier>`` term is a placeholder value you assign that represents
-an element of the array field name that prefixes it. This value must
-start with a lowercase letter and contain only alphanumeric characters.
+``arrayFilters`` object.
+
+The ``<identifier>`` value is a placeholder you assign that represents
+an element of the array field. This value must start with a lowercase
+letter and contain only alphanumeric characters.
```

Comment:
Suggestion:
Since the previous suggestion mentions omitting "value" from the description of the placeholder, I think this should provide some info on what the value pertains to. You can also avoid passive voice by using a sentence in this form. E.g.:

"You must assign a value to the placeholder that starts with a lowercase letter and contains only alphanumeric characters."